insights/parsers/mount.py

```diff
@@ -304,6 +328,75 @@ def _parse_mounts(self, content):
             self.mounts[mount['mount_point']] = entry
 
 
+@parser(Specs.mountinfo)
+class MountInfo(MountedFileSystems):
+    """
+    Class to parse the content of ``/proc/self/mountinfo`` file.
+
+    .. note::
+        Please refer to its super-class :class:`MountedFileSystems` for more details.
+
+    The typical content of ``/proc/self/mountinfo`` file looks like::
+
+        39 0 253:0 / / rw,relatime shared:1 - xfs /dev/mapper/rootvg-lvlocroot rw,attr2,inode64,noquota
+        47 39 8:1 / /boot rw,relatime shared:30 - xfs /dev/sda1 rw,attr2,inode64,noquota
+        65 39 253:19 / /data rw,relatime shared:44 - ext4 /dev/mapper/vgdata-lvdata rw,data=ordered
+        58 39 253:15 / /opt rw,relatime shared:45 - xfs /dev/mapper/rootvg-lvlocopt rw,attr2,inode64,noquota
+        61 39 253:17 / /home rw,nosuid,relatime shared:46 - xfs /dev/mapper/rootvg-lvlochome rw,attr2,inode64,noquotao
+
```

Comment:
It will be helpful to the rule developer if you can document the returned attributes.